nwb.config.js

```diff
@@ -0,0 +1,23 @@
+const browsers = require('./test/browsers');
```

Comment:
r we going to change build tools every month or so ? :P :D  :trollface:  next one --> https://neutrino.js.org/